cpp/include/mun/struct_ref.h

```diff
@@ -79,7 +81,7 @@ class StructRef {
      *
      * \return a pointer to the struct's type information
      */
-    const MunUnsafeTypeInfo info() const noexcept { return m_runtime->ptr_type(raw()); }
+    const TypeInfo info() const noexcept { return m_runtime->ptr_type(raw()); }
```

Comment:
The `const` here doesnt mean anything.

```suggestion
    TypeInfo info() const noexcept { return m_runtime->ptr_type(raw()); }
```